todo.js

```diff
@@ -16,53 +16,60 @@ Keep the responsibilities separated as best you can:
 5. Rather user input and taking the appropriate actions
 */
 let process = require('process');
-let command = process.argv[2];
-let secondCommand = process.argv[3];
-// takes the third argument because the first two are the node and the location of the file
 let fs = require('fs');
-
 // file system, reads files
 
-function openAndRead(filename) {
+function openAndReadFile(filename) {
   let text = fs.readFileSync(filename, 'utf-8');
   text = text.split('\n');
+  return text;
+}
+function printList(text) {
   console.log('Here is your list!');
   for (let i = 0; i < text.length - 1; i += 1) {
     console.log(` ${i + 1}. ${text[i]} `);
     // prints out the literal string, you put the data within the ${} when it is a variable and not a string
   }
 }
 
-function writingList() { // no parameter because command is on the command line
-  if (command === 'list') {
-    openAndRead('todos.txt');
-    // takes the information from the other function, any variables from there, edit that function
-  }
+function addToList(text, addition) {
+  console.log(`Appending '${addition}' to your TODO list`);
+  console.log();
+  fs.appendFileSync('todos.txt', addition);
+  // first parameter is the text file you add it to, next is what you're adding
```

Comment:
You're passing in the contents of the file, hard-coding the name of the file in the function to do the work, and then not using the first param anywhere.

Read my feedback on Will's code I linked elsewhere. The first version of his code was similar to yours.